### PR TITLE
Only prevent default on alt key

### PR DESCRIPTION
**Pull request by namse**
Related PR: #206 
Related Zulip: https://namseent.zulipchat.com/#narrow/stream/332914-.EA.B8.B0.EC.88.A0.EC.97.B0.EA.B5.AC.EC.86.8C/topic/Fix.20text-input.20space

I found that key-down's preventing default block input space and arrow keys. I fixed it.

Check list
- It still block alt+arrow key
- It doesn't block space and arrow key in text input

### CODE REVIEW

File: namui/src/namui/system/keyboard.rs [hunk 1]
```diff
@@ -45,17 +45,8 @@ impl KeyboardSystem {
                         let mut pressing_code_set = pressing_code_set.write().unwrap();
                         pressing_code_set.insert(code);
 
-                        match code {
-                            Code::Space
```
Comment (bigfoodK) [inline namui/src/namui/system/keyboard.rs:49]: Does scrolling occur when the space bar is pressed?

Comment (namse) [reply namui/src/namui/system/keyboard.rs:49]: I'm not sure why I did care about scrolling. We don't have scroll-able element, isn't it?

Comment (bigfoodK) [reply namui/src/namui/system/keyboard.rs:49]: Yes, we only use the canvas that fits exactly the size of the viewport

Comment (bigfoodK) [reply namui/src/namui/system/keyboard.rs:49]: Then I think it's ok to merge

Comment (namse) [reply namui/src/namui/system/keyboard.rs:49]: I think the main purpose of #206 is alt+left or alt+right.

**Review by bigfoodK — APPROVED** [review]

**Pull request merged**